src/Compilers/Core/Portable/CommandLine/AnalyzerConfig.cs

```diff
@@ -180,7 +179,7 @@ public static AnalyzerConfig Parse(SourceText text, string pathToFile)
             // Add the last section
             addNewSection();
 
-            return new AnalyzerConfig(globalSection, namedSectionBuilder.ToImmutable(), pathToFile);
+            return new AnalyzerConfig(globalSection!, namedSectionBuilder.ToImmutable(), pathToFile);
```

Comment:
It seems like `globalSection` could be `null` here in the case there wasn't a section. Am I missing how this always set?

Reply:
The above code always calls `addNewSection` at least once, and the first one will have `activeSectionName` set to blank.